packages/stark-ui/src/modules/input-mask-directives/directives/timestamp-mask.directive.ts

```diff
@@ -122,7 +122,8 @@ export class StarkTimestampMaskDirective extends MaskedInputDirective implements
 			return {
 				pipe: <any>createTimestampPipe(timestampMaskConfig.format),
 				mask: this.convertFormatIntoMask(timestampMaskConfig.format),
-				placeholderChar: "_"
+				placeholderChar: "_",
+				keepCharPositions: true // to avoid weird date values when deleting characters (see https://github.com/NationalBankBelgium/stark/issues/1260)
```

Comment:
It's not really working fine... If we have 13/05/2019 and we want to change the month for December, we cannot do it.
We can obtain 13/0_/2019 but not 13/__/2019... Maybe it's already better than nothing 😊

Reply:
As discussed, I've partially fixed this issue. There are still some edge cases that are not covered but I think they are not urgent for now so we can release soon.

So I've created #1277 to track the issue for those edge cases that we should fix later on 😉